code/modules/paperwork/paperplane.dm

```diff
@@ -110,9 +110,9 @@
 
 /obj/item/paper/examine(mob/user)
 	. = ..()
-	. += span_notice("Alt-click [src] to fold it into a paper plane.")
+	. += span_notice("Right click [src] to fold it into a paper plane.")
```

Comment:
Contextual screentips would be nice if you want to go the extra mile

Reply:
I do enjoy going the extra mile, I will do this.

Reply:
By "doing this now", I mean like "do it two weeks later", but at least I didn't forget.

Reply:
Actually, looking more into this, I don't see an existing example of the tooltips system interacting with mode changing for items when that mode changing involves holding it in your hand.

Health analyzers have tooltips for what mode they're in, but they don't have tooltips reminding you can switch the mode by attacking inhand...

Reply:
I'd appreciate insight if we want to progress with this, @timothymtorres

Reply:
The code I gave you on discord should give you a nice starting point to implement this.  If you need anymore help just ping me again.

Reply:
are we not gonna do screentips, it's marked as ready to review but no new commits

Reply:
No, I should do screentips, lemmie put it back to Draft 😞